src/main/java/seedu/address/model/question/Importance.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.model.question;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Question's importance in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidImportance(String)}
+ */
+public class Importance {
+
+
+    public static final String MESSAGE_CONSTRAINTS =
```

Comment:
as mentioned before, just a future note, this can be replaced by an enum. But for now it's ok